.github/workflows/sync_release_branch.yml

```diff
@@ -0,0 +1,27 @@
+name: Commits pushed to main
+on:
+  push:
+    branches:
+      - main
+jobs:
+  update_release_branch:
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/main'
+    steps:
+      - name: Update git # Apparently ubuntu-latest has an older version of git that causes some issues with scripts.
+        run: |
+          sudo add-apt-repository -y ppa:git-core/ppa
+          sudo apt-get update
+          sudo apt-get install git -y
+      - uses: actions/checkout@v2
+        with:
+          ref: releases/next
+          token: ${{ github.token }} # This will need to change to a token belonging to a service account.
+          fetch-depth: 0
+      - name: Merge changes from main
+        run: |
+          cd $GITHUB_WORKSPACE
+          git fetch --tags
+          git status
+          git merge origin/main
```

Comment:
What do we do if the merge fails?

Reply:
I think in case of conflict, this workflow would just fail and we would have to resort to manually merging the changes from main to releases/next. That's part of the reason why I ended up creating a separate workflow to sync the branches, instead of kicking off the entire versioning process from a single workflow. 

Ideally, the only changes made to the releases/next branch will be the updates to gradle.properties and changelog, so the risk of conflicting changes should be fairly low, but definitely something to keep an eye on and adjust.

Reply:
Could we prevent conflicts altogether by doing something like:
```
git checkout main
git branch -D release
git checkout -b release
git push -f release
```